src/SDKs/Monitor/Management.Monitor/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Monitor Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Monitor operations.")]
 
-[assembly: AssemblyVersion("0.20.0.0")]
-[assembly: AssemblyFileVersion("0.20.0.0")]
+[assembly: AssemblyVersion("0.20.1.0")]
```

Comment:
`AssemblyVersion` must always be the first major release version i.e. (0.9.0), only `AssemblyFileVersion` needs to be updated for non-major version updates

Reply:
@gucalder I missed that this is 0.x.x version, in that case your earlier change was correct. Apologies for the incorrect feedback.
Please revert both `AssemblyVersion` and `AssemblyFileVersion` to `0.20.1.0`